hugo-site/content/docs/introduction.md

```diff
@@ -70,3 +70,79 @@ Project Trident provides many features:
 -   **Easy system administration:** Project Trident provides many graphical tools for performing system administration.
 
 -   **Vibrant community:** Project Trident has a friendly and helpful community.
+
+### Security
+
+The Project Trident system is secure by default.
+This section is an overview of the built-in security features.
+There are also tips about increasing the security of the installed system beyond the configured defaults.
+
+The security features built into Project Trident include:
+
+- **Naturally immune to viruses and other malware written for other Operating Systems:** Most viruses are written to exploit the Windows operating system.
+  These are incompatible with the binaries and paths found on a Project Trident system.
+  Additional antivirus software is also available in the Appcafe.
+  This is useful when sending or forwarding email attachments to users running other operating systems.
+
+- **Potential for serious damage is limited:** Privilege separation between users and the administrator account (root) are built-in.
+  Files and directories can only be modified by root or users and groups with permission.
+  Any executed programs or scripts are only granted the permissions of that user.
+  A malicious program can only infect the files and directories owned by the user.
+  Core operating system files are protected.
+  Only users that are *wheel* and/or *operator* group members can gain administrative access.
+  These users are still not allowed to list directory contents or access files outside of the set "user" and "group" permissions.
+
+- **Built-in firewall:** The default firewall ruleset allows Internet access and any available network shares.
+  The firewall does not allow any inbound connections to the computer unless authorized by the user.
+
+- **Few default services:** All boot services can be viewed in the Service Manager.
+  Service Manager also allows starting, stopping, and adding or removing from boot any system service.
+
+- **SSH is disabled by default:** SSH can only be enabled by the administrator (root).
+  This prevents bots and outside individuals from accessing a Project Trident system.
+  If SSH access is required, add `sshd\_enable=YES` to **/etc/rc.conf**.
```

Comment:
This line needs to be deleted. The sshd_enable=YES lines are not needed with OpenRC

Reply:
Perhaps add a line detailing the `rc-update add sshd` command to use to enable it at bootup.